riscv/src/continuations.rs

```diff
@@ -424,7 +424,9 @@ pub fn rust_continuations_dry_run<F: FieldElement>(
             .map(|reg| {
                 let reg = reg.strip_prefix("main.").unwrap();
                 let id = Register::from(reg).addr();
-                *register_memory_snapshot.get(&(id as u32)).unwrap()
+                *register_memory_snapshot
+                    .get(&(id as u32))
+                    .unwrap_or(&0.into())
```

Comment:
this is fine i think, but i wonder is the executor itself should take care of this (zero out unused registers)

Reply:
for that it would have to know which registers we're using

Reply:
mmm right